etcdctl/ctlv3/command/get_command.go

```diff
@@ -137,6 +137,13 @@ func getGetOp(cmd *cobra.Command, args []string) (string, []clientv3.OpOption) {
 		ExitWithError(ExitBadFeature, fmt.Errorf("bad sort target %v", getSortTarget))
 	}
 
+	if sortByTarget != clientv3.SortByKey && sortByOrder == clientv3.SortNone {
+		// Since current mvcc.Range implementation returns results
```

Comment:
if any target is specified and sort_order == none, we should set sort order to ascend.